packages/inference/src/tasks/cv/imageToVideo.ts

```diff
@@ -0,0 +1,23 @@
+import type { ImageToVideoInput } from "@huggingface/tasks";
```

Comment:
Let's not add this file yet and wait for a follow-up PR. The current implementation assumes it works with FalAI and Replicate which is not the case yet -and not tested-. Better to separate them and keep this PR for specs definition only.